spruned/main.py

```diff
@@ -25,7 +25,7 @@
         headers_reactor.add_on_new_header_callback(blocks_reactor.start)
         headers_reactor.add_on_best_height_hit_volatile_callbacks(blocks_reactor.bootstrap_blocks)
         loop.create_task(headers_reactor.start())
-        loop.create_task(p2p_connectionpool.connect())
+        loop.create_task(p2p_interface.start())
```

Comment:
Yes, this is the proper entry point, thanks!
However, I dislike that the connectionpool is able to cast new peers from the file by itself, it's a discrepancy which masked the bug you've seen (I used to delete the storage folder and not the p2p peers list in my tests), I'll move the peers-from-file loader outside the connection pool.